pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/non/persistent/NonPersistentDispatcher.java

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Can we name the path as service/nonpersistent/ as one word in stead of two levels non/persistent ?